lib/rules/no-loop-func.js

```diff
@@ -9,10 +9,21 @@
 // Helpers
 //------------------------------------------------------------------------------
 
+const SKIPPED_IIFE_NODES = new Set();
```

Comment:
Since this Set is at the module level, it will hold references to the nodes, and indirectly to the entire AST which therefore cannot be removed from memory even after the file's linting is complete and its AST is no longer needed.

To avoid memory leaks, I think it's fine to move this Set and helper functions that are using it into `create(context) {}`.

Reply:
Updated 👍🏻